<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusteriFormu</class>
 <widget class="QWidget" name="MusteriFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>musteri bilgileri</string>
  </property>
  <widget class="QPushButton" name="btn_kaydet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>680</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>KAYDET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guncelle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>680</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>GÜNCELLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sil">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SİL</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>365</width>
     <height>545</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>AD</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="txt_ad">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>SOYAD</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="txt_soyad"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>TC Kimlik</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="txt_tc"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Doğum Tarihi</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QDateEdit" name="date_dogum">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Adres</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QTextEdit" name="txt_adres"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Ev Telefonu</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="txt_telefon_ev"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Cep Telefonu</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="txt_telefon_cep"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Mesleği</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="txt_meslek"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Ehliyet Sınıfı</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QComboBox" name="cmb_ehliyet"/>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Medeni Durumu</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QComboBox" name="cmb_medeni"/>
    </item>
    <item row="10" column="2">
     <widget class="QComboBox" name="cmb_egitim"/>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Eğitim Durumu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>92</y>
     <width>1011</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string> ARAMA :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_arama"/>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>MÜŞTERİ LİSTESİ :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tbl_liste"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
